### ISSUE TITLE
!ATTENTION! Stop Sniping, that's why some accounts are not able to login anymore...

### ISSUE THREAD

**Issue by iAmShorty** [NECROBOTIO/NecroBot#5454]
like the subject tells you, stop sniping. i've tested with 3 new accounts, two with active sniping are not able to login anymore, the third works fine for now.

**Comment by kurozani** [comment]
I snipe since 3 hours ago, from level 1, @ leevl 20 now

doing fine (I hope)

**Comment by venduss** [comment]
I second that. I couldn't visit more than 1 pokestop while sniping was enabled. After I disabled it, my bot started to work properly.

**Comment by john-hardesky** [comment]
Most likely has nothing to do with that but rather the login issue occurs because Google login US has a critical flag on the Pokemon Go servers ([ispokemongodownornot.com](http://ispokemongodownornot.com/)).
I'm using 2 different accounts (Google EU) and they work fine.

**Comment by headhunter1215** [comment]
@iAmShorty  i didn't use sniping and i get [failed to get game data from server] when i try to login and load game on 2 accounts that din't ever use snipe.  @john-hardesky maybe.. but i can't login on 1 ptc and 1 google account that used bot BUT 1 other ptc acc that didn't use necro , i can login ...  doesn't look to good if you ask me :/

**Comment by shawly** [comment]
Well I disabled sniping and now it says "no usable Pokestops found is your maximum distance too small?"

**Comment by venduss** [comment]
@shawly You're softbanned I presume.

**Comment by BigPhalus** [comment]
@venduss @shawly its not a softban if it was the bot would still see the pokestops, it just wont be able loot anything from them

**Comment by john-hardesky** [comment]
I logged into my main account (never used a bot on it) and I can't see any Pokestops.
Probably a Pokemon Go server issue.

**Comment by headhunter1215** [comment]
@john-hardesky yeah you're probably safe if you can get into the game any other stuff seems like niantic problems however i can't get in with 2 of my account's that used necrobot and i fear for the worst.

**Comment by michael8228** [comment]
@headhunter1215 isit failed to get game data from the server?

**Comment by headhunter1215** [comment]
@michael8228 yes that's the error i get on those accounts

**Comment by michael8228** [comment]
@headhunter1215 it's annoying! have you fix yours?

**Comment by headhunter1215** [comment]
@sianyuw7  @michael8228  
no, i haven't fixed it , i think this can't be "fixed" it could be bad luck with niantic server but it feels like a ban.
To me the question is, is it a new (longer) soft ban, or is it perm ban.

**Comment by sianyuw7** [comment]
thanks for info @headhunter1215

**Comment by shawly** [comment]
Alright I just started the bot again after an hour and now it works fine.
